ldap/servers/slapd/back-ldbm/db-bdb/bdb_import.c

```diff
@@ -2764,7 +2765,13 @@ bdb_back_ldif2db(Slapi_PBlock *pb)
     }
 
     /* old style -- do it all synchronously (THIS IS GOING AWAY SOON) */
-    return import_main_offline((void *)job);
+	ret = import_main_offline((void *)job);
+	if(job->skipped) {
+		int warn = 2;
```

Comment:
Rather than the assuming warn=2 here, it would be good to have a warning enum type that can be extended in the future with different warning types that could exist in the future. Enum is preferable over a #define as well because that way there is type checking that can exist over the usage of the value.

Reply:
Great idea